app/Http/Controllers/Dashboard.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Services\BoardService;
+use App\Services\ProfileService;
+use App\Services\SessionService;
+use Carbon\CarbonInterval;
+
+class Dashboard extends Controller
+{
+    private SessionService $sessionService;
+
+    private BoardService $boardService;
+
+    private ProfileService $profileService;
+
+    public function __construct()
+    {
+        $this->sessionService = new SessionService();
+        $this->boardService = new BoardService();
+        $this->profileService = new ProfileService();
+    }
+
+    //Method to show dashboard and send necessary information
```

Comment:
```suggestion
    /**
    * Method to show dashboard and send necessary information
    */
```
Single line comments like that are generally not used for functions, they also don't show up on IDEs when you search the docs for a method.